libs/sdk-ui-ext/src/internal/translations/en-US.json

```diff
@@ -576,6 +576,21 @@
         "limit": 0,
         "translate": false
     },
+    "dashboard.bucket.category_columns_warning._measure": {
+        "value": "To add columns, total items from measures or rows should not be higher than 20",
```

Comment:
can we use placeholder here too, smth like {oldLimit} ?

Reply:
Yes, added it.